tachyon/crypto/sumcheck/BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+load("//bazel:tachyon_cc.bzl", "tachyon_cc_library")
+
+package(default_visibility = ["//visibility:public"])
+
+tachyon_cc_library(
+    name = "sumcheck_proving_key",
+    hdrs = ["sumcheck_proving_key.h"],
+    deps = [
+        ":sumcheck_verifying_key",
+        "//tachyon/base:random",
+        "//tachyon/base/buffer:copyable",
```

Comment:
It's up to you, though. You can remove this dependencies because `sumcheck_verifying_key` depends on them